homeassistant/components/logbook.py

```diff
@@ -37,19 +39,19 @@
 
 ATTR_NAME = 'name'
 ATTR_MESSAGE = 'message'
-ATTR_DOMAIN = 'domain'
+ATTR_DOMAIN_INPUT = 'component'
```

Comment:
We should keep the same naming as the `log_entry` method. You can just use `ATTR_DOMAIN`